custom/icds_reports/static/js/directives/adolescent-girls/adolescent-girls.directive.js

```diff
@@ -16,7 +16,7 @@ function AdolescentWomenController($scope, $routeParams, $location, $filter, dem
     vm.data = {
         legendTitle: 'Number of Women',
     };
-    vm.filters = ['age', 'gender'];
+    vm.filters = ['age', 'gender', 'data_period'];
```

Comment:
filters array used to get the filters that are not to be displayed on filters page. Since we need data_period filter only in ppd, I had to add it in all other files

Reply:
ow. this is so weird to have this not check. But It feels its not you who introduced this.

Reply:
Yeah, it was developed that way from the start